cfme/tests/containers/test_containers_default_views.py

```diff
@@ -43,9 +52,10 @@ def test_containers_providers_default_view(button_group, view):
               to Grid/Tile/List view
             * Goes to Compute --> Containers --> Providers and verifies the selected view
         """
-    sel.force_navigate('my_settings_default_views')
+    navigate_to(Server, 'MySettings')
+    tabs.select_tab("Default Views")
     bg = ButtonGroup(button_group)
     bg.choose(view)
     sel.click(form_buttons.save)
-    sel.force_navigate(mapping[button_group])
+    navigate_to(mapping[button_group], 'All')
```

Comment:
The ```use_resetter``` option was just added to the code base, you should give it a try here. You should also have a finalizer that sets them back to their defaults.
```python
navigate_to(mapping[button_group], 'All', use_resetter=False)
```